app/src/main/java/fr/neamar/kiss/result/SettingsResult.java

```diff
@@ -53,6 +53,7 @@ public void doLaunch(Context context, View v) {
         if (!settingPojo.packageName.isEmpty()) {
             intent.setClassName(settingPojo.packageName, settingPojo.settingName);
         }
+		intent.setSourceBounds(v.getClipBounds());
```

Comment:
I thought I had fixed that dumb possible error source, but apparently I only did so in `AppResult`.

I've already fixed it locally.